src/jsglue.cpp

```diff
@@ -849,18 +804,11 @@ static size_t MallocSizeOf(const void* aPtr)
 }
 
 bool
-CollectServoSizes(JSRuntime *rt, JS::ServoSizes *sizes, GetSize gs)
+CollectServoSizes(JSContext* cx, JS::ServoSizes *sizes)
 {
-  mozilla::PodZero(sizes);
-
-  ServoDOMVisitor sdv(gs, ShouldMeasureObject);
-
-  return JS::AddServoSizeOf(rt, MallocSizeOf, &sdv, sizes);
-}
-
-void
-InitializeMemoryReporter(WantToMeasure wtm){
-  gWantToMeasure = wtm;
+    mozilla::PodZero(sizes);
+    return JS::AddServoSizeOf(cx, MallocSizeOf,
+                              /* ObjectPrivateVisitor = */ nullptr, sizes);
```

Comment:
I'm concerned that we're regressing the DOM measurement.

Reply:
A fair concern. I;ll have a look to see if there's a replacement for ServoDOMVisitor.